client/components/domains/domain-registration-suggestion/utility.js

```diff
@@ -0,0 +1,63 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { includes } from 'lodash';
+
+export const newTLDs = [
+	'.art',
+	'.bar',
+	'.beer',
+	'.buzz',
+	'.cab',
+	'.casa',
+	'.click',
+	'.coffee',
+	'.cooking',
+	'.design',
+	'.fashion',
+	'.fishing',
+	'.fit',
+	'.garden',
+	'.gift',
+	'.golf',
+	'.group',
+	'.help',
+	'.horse',
+	'.hospital',
+	'.ink',
+	'.jetzt',
+	'.link',
+	'.lol',
+	'.miami',
+	'.mom',
+	'.money',
+	'.movie',
+	'.network',
+	'.photo',
+	'.pics',
+	'.rest',
+	'.rodeo',
+	'.sexy',
+	'.style',
+	'.surf',
+	'.tattoo',
+	'.vip',
+	'.vodka',
+	'.wedding',
+	'.wiki',
+	'.work',
+	'.wtf',
+	'.yoga',
+];
+
+export const testTLDs = [ '.de' ];
+
+export function isNewTLD( tld ) {
```

Comment:
The `TLD` parts in this file should be `Tld` according to [our naming guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#naming-conventions):
```
// Bad
let userIDToDelete;
let siteURL;
```

```
// Good
let userIdToDelete;
let siteUrl;
```

So, `testTlds`, instead of `testTLDs`. Same for `isKrackenUI`. Again, nitpicking, but I like consistency :)

Reply:
Oh yeah, good point. I'll fix this.